DESIGN.md

```diff
@@ -36,28 +36,28 @@ OSM ships out-of-the-box with all necessary components to deploy a complete serv
 
 
 ## OSM Components & Interactions
-![OSM Components & Interactions](/docs/development_guide/osm-components-and-interactions.png)
+![OSM Components & Interactions](/docs/images/osm-components-and-interactions.png)
```

Comment:
Once the page is removed, this reference can be removed as well

Reply:
This is the link to the image, rather than the removed page. We will keep this link.